src/main/java/vg/civcraft/mc/civmodcore/CivStrings.java

```diff
@@ -0,0 +1,6 @@
+package vg.civcraft.mc.civmodcore;
+
+public final class CivStrings {
```

Comment:
good point. I was going to maybe add other strings to it but then didn't go that route. I'll remove.